docs/design_decisions.md

```diff
@@ -119,7 +112,11 @@ _**Ratio**_
 
 We willen "tight coupling" met andere bronnen voorkomen. Over de domeinen heen wordt alleen met links verwezen. 
 
-### DD3.3 We nemen geen (inverse) relaties uit een ander domein op
+_**Kanttekening**_
+
+Er zijn grofweg twee categoriën Hal-links waar we gebruik van maken. Links naar resources binnen het eigen domein en links naar resources die in een ander domein beheerd worden. Om discoverability te bereiken, worden voor beide categorieën de Hal-link opgenomen naar de gerelateerde resource. 
```

Comment:
Dit komt nu een beetje uit de lucht vallen, want het punt gaat over embedded van gerelateerde resources.

Bedoel je zoiets:
> * Indien mogelijk wordt de gebruiker ook de mogelijkheid geboden de gerelateerde resource uit dezelfde bron op te halen zonder gebruik te maken van deze te embedden. Hiervoor wordt een HAL-link opgenomen naar de gerelateerde resource.
> * Een gerelateerde resource in een andere bron kan ook discoverable worden gemaakt door het opnemen van een HAL-link naar die externe resource. De gebruiker kan deze dan, met behulp van de geleverde uri, zelf ophalen bij deze andere bron.

Reply:
Het fragment is niet van mijn hand, ik vermoed dat Johan het heeft geschreven dus je moet hem eigenlijk vragen wat hij er mee bedoelt. Ik denk echter wel dat de bullets zijn bedoelingen duidelijk maken.